README.md

```diff
@@ -7,7 +7,7 @@ The server itself is implemented in [Ruby](https://www.ruby-lang.org/en/) using
 The server logs all API requests it performs to the terminal, so you can see what's going on even without diving into the code.
 
 ## Integration with the Klarna on Demand API
```

Comment:
I guess "on demand" is kind of a single word. So "on Demand" seems odd to me. Ben & Case probably now better.